test/unit/spec/appWidgets/view/dynamicDropdownViewSpec.js

```diff
@@ -7,15 +8,37 @@ define(['widgets/appWidgets2/view/dynamicDropdownView', 'common/runtime', 'testU
 
     const DEFAULT_VALUE = 'banana';
     const SELECTOR = 'select.form-control[data-element="input"]';
+    const SELECT2_OPTION_SELECTOR = '.select2-container .select2-selection__rendered';
```

Comment:
This test spec needed some editing to adjust to the selectors and such of the input version, which actually makes the testing more precise.. The input one was also finicky about having a more complete parameter spec.